packages/block-editor/src/components/index.js

```diff
@@ -22,6 +22,7 @@ export { default as __experimentalGradientPickerControl } from './gradient-picke
 export { default as InnerBlocks } from './inner-blocks';
 export { default as InspectorAdvancedControls } from './inspector-advanced-controls';
 export { default as InspectorControls } from './inspector-controls';
+export { default as LinkControl } from './link-control';
```

Comment:
Lets make this experimental
```suggestion
export { default as __experimentalLinkControl } from './link-control';
```

Reply:
✅ Done